.github/workflows/embedding-sdk-component-tests.yml

```diff
@@ -30,6 +30,19 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Prepare Docker containers
+        uses: ./.github/actions/e2e-prepare-containers
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          maildev: true
```

Comment:
Since we call `H.setupSMTP();` to need to ensure we run this SMTP server for SDK tests.